packages/task/src/browser/task-service.ts

```diff
@@ -111,7 +111,7 @@ export class TaskService implements TaskConfigurationClient {
         // notify user that task has started
         this.taskWatcher.onTaskCreated((event: TaskInfo) => {
             if (this.isEventForThisClient(event.ctx)) {
-                this.messageService.info(`Task #${event.taskId} created - ${event.config.label}`);
+                this.messageService.info(`Task ${event.config.type}: ${event.config.label} has been started`);
```

Comment:
an issue with this line is that, contributed (aka. detected) tasks' types are overwritten. e.g., `npm: install` whose type should be `npm`, will be displayed as `shell: install` here. Maybe we should differentiate the contributed (aka. detected) tasks from the others?

this comment also applies to the task exit event.

Reply:
![Peek 2019-05-30 09-03](https://user-images.githubusercontent.com/37082801/58634722-07543d80-82ba-11e9-9e89-ad7947771456.gif)

Reply:
I'll take a look on it.

Reply:
Fixed